frontend_lib/src/css/Variable.styl

```
@@ -27,12 +27,22 @@ zIndexOverHeader = 50
 headerHeight = 61px // INFO - CH - 2019-04-19 - the total header height. Used to calculate height of layout like height: "calc(100% - %s)" % headerHeight
 sidebarWidth = 300px
 
-fontSizeTitle = 25px
-fontSizeMedium = 20px
-fontSizeSmall = 14px
+fontSizeTitle = 20px
+fontSizeStandard = 16px
+fontSizeMetadata = 14px
+
 listItemHeight = 5px
 popupWidth = 420px
-bigIconSize = 30px
+bigIconSize = 20px
+
+borderRadiusStandard = 5px
```

```suggestion
standardBorderRadius = 5px
```
To be consistent with `standardSpacing`.

---

will fix